Update MS teams docs (#32395)
demisto/content: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_description.md

Use the Microsoft Teams integration to send messages and notifications to your team members and create meetings. 

> Note: the integration is supported in Cortex XSOAR 8 without using an engine.

Reviewer: ```suggestion
Note: This integration is supported in Cortex XSOAR 8 when not using an engine.
```
